src/parser/bison_parser.y

```diff
@@ -725,10 +747,13 @@ opt_having:
 	|	/* empty */ { $$ = nullptr; }
 
 opt_order:
-		ORDER BY order_list { $$ = $3; }
+		order { $$ = $1; }
 	|	/* empty */ { $$ = nullptr; }
 	;
 
+order:
```

Comment:
Well yeah, that was a left over from one of our first tries that I did not remove for this pr. TBH I almost feel like it would help the understanding for people to have a difference here. Will talk about it with our TA on wednesday

Reply:
Fixed in new patch